microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -6,5 +6,143 @@ include::_attributes/common-attributes.adoc[]
+<7> unit. The number of stripes in the logical volume.
+<8> String. The amount of data that is written to one device before moving to the next device. 
+<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
++
+[NOTE]
+====
+Striping can be configured by using the dedicated options (`stripe` and `stripe-size`) and `lvcreate-options`. Either option can be used, but they cannot be used together. Using `stripe` and `stripe-size` with `lvcreate-options` leads to duplicate arguments to `lvcreate`. You should never set `lvcreate-options: ["--stripes=n"]` and `stripe: n` at the same time. You can, however, use both, when `lvcreate-options` is not used for striping. For example: 
+
+[source,yaml]
+----
+stripe: 2
+lvcreate-options: ["--mirrors=1"]
+----
+====
+
+[id="setting-lvms-path"]
+=== Setting the LVMS path 
+
+The `config.yaml` file for the LMVS should be written to the same directory as the MicroShift `config.yaml` file. If a MicroShift `config.yaml` file does not exist, MicroShift will create an LVMS `config.yaml` file and automatically populate the configuration fields with the default settings. The following paths are checked for the `config.yaml` file, depending on the user that MicroShift is run by: 
+
+
+.LVMS paths 
+[options="header",cols="1,3"]
+|===
+| MicroShift user | Configuration directory 
+|Global administrator | `/etc/microshift/lvmd.yaml`
+|===
+
+[id="lvms-system-requirements"]
+== LVMS system requirements 
+
+{product-title}'s LVMS requires the following system specifications. 
+
+[id="lvms-volume-group-name"]
+=== Volume Group Name 
+
+The default integration of LVMS assumes a volume group named `rhel`. Prior to launching, the `lvmd.yaml` configuration file must specify an existing volume group on the node with sufficient capacity for workload storage. If the volume group does not exist, the node controller will fail to start and enter a `CrashLoopBackoff` state. 
+
+[id="lvms-volume-size-increments"]
+=== Volume size increments 
+
+The LVMS provisions storage in increments of 1 GB. Storage requests are rounded up to the nearest gigabyte (GB). When a volume group's capacity is less than 1 GB, the `PersistentVolumeClaim` registers a `ProvisioningFailed` event, for example: 
+
+[source,terminal]
+----
+Warning  ProvisioningFailed    3s (x2 over 5s)  topolvm.cybozu.com_topolvm-controller-858c78d96c-xttzp_0fa83aef-2070-4ae2-bcb9-163f818dcd9f failed to provision volume with 
+StorageClass "topolvm-provisioner": rpc error: code = ResourceExhausted desc = no enough space left on VG: free=(BYTES_INT), requested=(BYTES_INT)
+----
+
+[id=using-lvms]
+== Using the LVMS
+
+The LVMS `StorageClass` is deployed with a default `StorageClass`. Any `PersistentVolumeClaim` objects without a `.spec.storageClassName` defined automatically has a `PersistentVolume` provisioned from the default `StorageClass`. 
+
+Use the following procedure to provision and mount a logical volume to a pod. 
+
+.Procedure 
+
+* Enter...
```

Comment:
I think if you keep it a procedure with one step, it should be updated:

```suggestion
.Procedure 

* To provision and mount a logical volume to a pod, enter the following command:
```
ISG: [Commands, parameters, and options in instructions](https://www.ibm.com/docs/en/ibm-style?topic=elements-commands#commands-parameters-and-options-in-instructions)

Alternatively, it might make sense to take it another direction, such as: 
```suggestion

The following example demonstrates how to provision and mount a logical volume to a pod. 
```
ISG: [Code and command examples](https://www.ibm.com/docs/en/ibm-style?topic=elements-code-command-examples)